great-idea-website/index.html

```diff
@@ -15,22 +15,19 @@
 </head>
 
   <body>
+<div class="container">
 <header>
   <nav class="menubar">
     <ol>
-      <li>Services</li>
-      <li>Productt</li>
-      <li>Vision</li>
-      <li>About</li>
-      <li>Contact</li>
+      <li class="menuItem">Services</li>
+      <li class="menuItem">Productt</li>
+      <li class="menuItem">Vision</li>
+      <li class="menuItem">About</li>
+      <li class="menuItem">Contact</li>
     </ol>
```

Comment:
for the element in the nav it will be better to use <a></a> tag instead of a <ol> <li></li> </ol>